libs/payments/stripe/src/lib/accountCustomer/accountCustomer.manager.ts

```diff
@@ -65,15 +65,6 @@ export class AccountCustomerManager {
     }
   }
 
-  public async getStripeCustomerIdByUid(uid: string) {
-    try {
-      const { stripeCustomerId } = await this.getAccountCustomerByUid(uid);
-      return stripeCustomerId;
-    } catch (error) {
-      return null;
-    }
-  }
-
```

Comment:
Removed since this is just a wrapper of `getAccountCustomerByUid`. We should consume that instead.